### ISSUE TITLE
Agregar funcionalidades offline

### ISSUE THREAD

**Issue by juaneladio** [AngelFQC/museaMoz#12]
Evaluar posibilidad de incorporar AppCaché o algún mecanismo que permita almacenar localmente parte de la aplicación durante la primera ejecución.

**Comment by AngelFQC** [comment]
Sí, en eso estoy. Espero que para el fin de semana actualice nueva versión
